app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power,
+                 average_rating, count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars):
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def wash_single_car(self, car):
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def calculate_washing_price(self, car):
+        if car.clean_mark < self.clean_power:
+            income1 = car.comfort_class * (
+                self.clean_power - car.clean_mark) * \
+                self.average_rating / self.distance_from_city_center
```

Comment:
the same here

Reply:
What do you mean. divide the term into objects

Reply:
split this huge expression, for the variables